src/game/client/draw/w3dtreedraw.cpp

```diff
@@ -0,0 +1,117 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D Tree Draw
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "w3dtreedraw.h"
+#include "baseheightmap.h"
+#include "drawable.h"
+
+W3DTreeDrawModuleData::W3DTreeDrawModuleData() :
+    m_moveOutwardTime(1),
+    m_moveInwardTime(1),
+    m_moveOutwardDistanceFactor(1.0f),
+    m_darkeningFactor(0.0f),
+    m_toppleFX(nullptr),
+    m_bounceFX(nullptr),
+    m_killWhenFinishedToppling(true),
+    m_doTopple(false),
+    m_doShadow(false),
+    m_initialVelocityPercent(0.2f),
+    m_initialAccelPercent(0.01f),
+    m_bounceVelocityPercent(0.3f),
+    m_minimumToppleSpeed(0.5f),
+    m_sinkTime(300),
+    m_sinkDistance(20.0f)
+{
+}
+
+void W3DTreeDrawModuleData::Build_Field_Parse(MultiIniFieldParse &p)
+{
+    static FieldParse dataFieldParse[] = {

```

Comment:
could you turn off clang format and align this more nicely.

Reply:
fixed.